peers/Cargo.toml

```diff
@@ -27,6 +27,7 @@ crossbeam-channel = "0.3"
 ed25519 = { version = "1.0.1", default-features = false, features = ["serde"] }
 failure = "0.1.5"
 grpcio = "0.6.0"
+mockall = "0.7.2"
```

Comment:
This moved from dev_dependencies to dependencies so that other crates can use mocks defined here in their tests. I think this is a limitation of cargo: if crate A depends on crate B, building A with the `test` attribute does not build B with the `test` attribute. 

I think this is OK, but would be happy to do something else if you've got an idea.